start_ocean.sh

```diff
@@ -63,10 +64,14 @@ export DB_CLIENT_KEY=""
 export DB_CLIENT_CERT=""
 CHECK_ELASTIC_VM_COUNT=true
 
+export BRIZO_LOG_LEVEL="INFO"
+export EVENTS_HANDLER_LOG_LEVEL="INFO"
+
 # Set a valid parity address and password to have seamless interaction with the `keeper`
 # it has to exist on the secret store signing node and as well on the keeper node
-export PUBLISHER_ADDRESS=0x068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0
-export PUBLISHER_PASSWORD=secret
+export PUBLISHER_ADDRESS=0x00bd138abd70e2f00903268f3db08f2d25677c9e
```

Comment:
i used `0x068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0` on purpose because it is not the "publisher account" in barge. `0x00bd138abd70e2f00903268f3db08f2d25677c9e` is used for many many different other things, including running some parts of the network itself.

Reply:
maybe we can just create a completely fresh account and use that only exclusively in barge?

Reply:
We just need a reasonable default that is available at least on spree. When running with other networks the user could specify the account of their choice (this is also possible on spree).

Reply:
Reverted back to `0x068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0`, also added the keyfile for this account in its own folder.